### ISSUE TITLE
Bootlooping due to overheating? (Possibly, hasn't been replicated yet)

### ISSUE THREAD

**Issue by cgarcia2097** [torontomulibrary/DMEPrinting#3]
Hi guys. Today I encountered an issue with the Taz's RPi where it continuously boot loops. By that I mean it goes through a normal boot up phase, but somehow finds itself too hot to continue and so restarts the system over and over again.

As for the event that caused the RPi to overheat, I would not know, but I think cooling fans and heat sinks on these RPi's are of strong recommendation as upgrades; says me.

P.S. Though I am an amateur with overclocking settings, I know for a fact the RPi's clocks are bone stock at 600MHz using the Octoprint instance. I have yet to do a stress test at these stock clocks to see the temperatures to which these Pis are operating. 

In short, if anyone has time, mind SSH'ing into an active machine and observe the temperatures?

**Comment by DongerZonie** [comment]
I think you may be over thinking it, start by trying to remove and reinsert the SD card.
Then try the SD card in a different SD card.
The failure is most likely there

**Comment by cgarcia2097** [comment]
The RPi boots fine... I swapped the RPi it was using (same SD card, different Pi) and the Taz is working again albeit a different IP address. My issue is the fact it overheated in the first place, and the Pi kept trying to boot into Octoprint without success. The temperature icon shows up while the Pi was booting.

**Comment by DongerZonie** [comment]
Closed because the rPi is operating normally
